src/wui/attack_window.cc

```diff
@@ -556,9 +558,6 @@ std::vector<Widelands::Serial> AttackPanel::soldiers() const {
 	return result;
 }
 
-constexpr int kSoldierIconWidth = 32;
-constexpr int kSoldierIconHeight = 30;
```

Comment:
These shouldn't be removed, only moved up or into some header to be usable in the constructor above.